components/Product/basic_usage.rst

```diff
@@ -0,0 +1,163 @@
+Basic Usage
+===========
+
+Creating a product
+------------------
+
+.. code-block:: php
+
+   <?php
+
+   use Sylius\Component\Product\Model\Product;
+
+   $product = new Product();
+
+   $product->getCreatedAt(); // returns the \DateTime when it was created
```

Comment:
We treat comments as sentences, so please use capital first letter and a dot at the end. :)